apps/modernization-ui/src/apps/search/patient/result/patientSearchResult.tsx

```diff
@@ -37,8 +37,29 @@ const displayAddresses = (result: PatientSearchResult): JSX.Element => (
 
 const displayPhones = (result: PatientSearchResult): string => result.phones.join('\n');
 const displayEmails = (result: PatientSearchResult): string => result.emails.join('\n');
+const displayPatientName = (result: PatientSearchResult): JSX.Element => (
+    <div>
+        <div>
+            <b>Legal name</b>
+            <br />
+            {result.legalName && displayName('fullLastFirst')(result.legalName)}
+
+            {result.names
+                .filter((v) => JSON.stringify(v) !== JSON.stringify(result.legalName))
+                .reverse()
+                .map((name, index) => (
+                    <div key={index}>
```

Comment:
Use the new <ItemGroup> element in design system here. It already has the right CSS styling per the Figma design.

Reply:
Also get latest from main and see `displayOtherNames `function. There is some overlap here.